dataset/src/test/scala/frameless/ops/PivotTest.scala

```diff
@@ -8,19 +8,19 @@ import org.scalacheck.Prop._
 import org.scalacheck.{Gen, Prop}
 
 class PivotTest extends TypedDatasetSuite {
-  def withCustomGenX4: Gen[Vector[X4[String, String, Int, Boolean]]] = {
```

Comment:
Was there an issue with `Int`?

Reply:
I've just retested it and it's working, previously it was not due to interim changes on the ExpressionEncoder integration, reverted.